### DISCUSSION TITLE
Google analytics not working

### DISCUSSION THREAD

**Discussion by xiaohsun** [tomfran/typo#54] [Q&A] [answered]
Hi everyone,

I added my Measurement ID to the hugo.toml file, but Google Analytics still isn't working after 48 hours. It indicates that my website doesn't include the Measurement ID.

Has anyone else experienced this problem?

**Comment by tomfran** [comment]
Try to inspect the built page on the browser to see if your token is there, as far as I know it should be for it to be tracked.

If it's not, it's probably due to the if statement on line 37:

https://github.com/tomfran/typo/blob/main/layouts/partials/head.html

This might be helpful (maybe it's outdated tho):
https://dev.to/tylerlwsmith/check-if-a-hugo-site-is-in-development-or-production-2beb

**Reply by xiaohsun** [reply]
Thank you for your response.

I discovered that {{ template "_internal/google_analytics.html" . }} isn’t working. However, if I replace it with the code provided by Google Analytics, it works correctly. This confirms that the issue is not related to the if statement.

I’m still trying to figure out the cause of this issue, but I really appreciate your help!

**Reply by xiaohsun** [reply, marked as answer]
I finally succeeded!

I moved googleAnalytics = "G-XXXXXXXX" before [taxonomies] in hugo.toml, and it worked perfectly.
I think this might be a bug or an issue of some kind.

Anyway, thank you so much for your help!

**Reply by JimmyRespawn** [reply]
> I finally succeeded!
> 
> I moved googleAnalytics = "G-XXXXXXXX" before [taxonomies] in hugo.toml, and it worked perfectly. I think this might be a bug or an issue of some kind.
> 
> Anyway, thank you so much for your help!

Ye, this fix for me as well. Idk why they stil donn't fix this.

**Reply by tomfran** [reply]
> Idk why they stil donn't fix this

You are more than welcomed and encouraged to open a PR with a fix you have already implemented.
